src/core/Table/Table.tsx

```diff
@@ -375,13 +375,13 @@ export const Table = <
           );
         })}
       </div>
-      <div {...getTableBodyProps({ className: 'iui-tables-body' })}>
+      <div {...getTableBodyProps({ className: 'iui-table-body' })}>
         {data.length !== 0 &&
```

Comment:
Can you set `min-width: undefined` on line 178 so it would take `min-width` from CSS?
![image](https://user-images.githubusercontent.com/36186912/121858037-ddcea900-ccfe-11eb-97ed-2baa008a41bd.png)

Reply:
and what about line 205?

Reply:
and what about maxwidth and width?